docs/superpowers/specs/2026-05-30-ios-dev-build-testing-design.html

```
@@ -0,0 +1,82 @@
+<ul><li>Launch the dev client, run <code>npm run start:device</code>, connect the phone to Metro.</li><li>Log in (remote Supabase), then exercise at least one agent feature (e.g. outfit suggest or chat) to confirm the phone → Cloud Run round-trip works end-to-end.</li></ul>
+</li>
+</ol>
+<h2>Out of scope / noted</h2>
+<ul>
+<li><strong>CORS:</strong> does not apply to native clients (no browser <code>Origin</code> enforcement). The backend&#x27;s existing CORS middleware is web-only; no change.</li>
+<li><strong>Cold starts:</strong> Cloud Run may add first-call latency; not addressed here.</li>
+<li><strong>TestFlight / production signing:</strong> explicitly out of scope — dev build only.</li>
+<li><strong>Native permissions:</strong> mic/location/photos/camera/speech usage strings are already declared in <code>app.json</code> plugin config; iOS prompts on first use, nothing to wire.</li>
+</ul>
+<h2>Success criteria</h2>
+<ul>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </ul> ], start tag match failed [ <ul> ] on line 77.

(tag-pair)